examples/nlp/gpt/conf/training_rm.yaml

```
@@ -113,7 +113,8 @@ model:
     reset_attention_mask: False # Reset attention mask after end-of-document token
     eod_mask_loss: False # Mask loss for the end of document tokens
     index_mapping_dir: null # path to save index mapping .npy files, by default will save in the same location as data_prefix
-    data_prefix: null
+    data_prefix: null # For multiple validation sets, use a dictionary that includes one key named `validation`
+    # (main validation set) and extra keys of the form `validation*` (secondary validation sets).
```

This format of comment looks a bit weird to me (continuing an inline comment on the next line) => suggestion
```suggestion

    # For multiple validation sets, use a dictionary that includes one key named `validation` (main
    # validation set) and extra keys of the form `validation*` (secondary validation sets), for instance:
    # model.data.data_prefix="{train:[train.jsonl],validation:[valid.jsonl],validation_set2:[valid2.jsonl],test:[test.jsonl]}"
    data_prefix: null 
```